recipes/sos-notebook/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f3ae4371f2a9cf493c77190010e73270ddc658904618937889ce14c0ecebb09
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
+
+requirements:
+  host:
+    - ipython
+    - notebook >=5.0.0
+    - pip
+    - python >=3.6
+  run:
+    - ipykernel
+    - ipython
+    - markdown
+    - matplotlib
+    - nbconvert >=5.1.1
+    - nbformat
+    - notebook >=5.0.0
+    - pandas
+    - python >=3.6
+    - setuptools
```

Comment:
`setuptools` is not needed for `run`.

Reply:
Then should I move it to `host`? It is called in `setup.py`, so I assume it is needed for installation.

https://github.com/vatlab/sos-notebook/blob/master/setup.py#L11